yb-voyager/src/tgtdb/target_db_interface.go

```diff
@@ -43,6 +43,7 @@ type TargetDB interface {
 	ImportBatch(batch Batch, args *ImportBatchArgs, exportDir string, tableSchema map[string]map[string]string, isRecoveryCandidate bool) (int64, error)
 	QuoteAttributeNames(tableNameTup sqlname.NameTuple, columns []string) ([]string, error)
 	GetPrimaryKeyColumns(table sqlname.NameTuple) ([]string, error)
+	GetPrimaryKeyConstraintName(tableNameTup sqlname.NameTuple) (string, error)
```

Comment:
consider implementing this inside YB ONLY and not expose it via the interface. Reasons being:
- We don't really need this for other DBs now. The interface has become too huge already. In fact, the outer layer (FileTaskImporter) doesn't really need to know what the PKColumns/PKConstraintName exactly are, right? We seem to be just getting this value and simply passing it back to tdb?
- It's only needed for the fast path, which is an internal detail of the yb.ImportBatch. 
- It seems a bit weird that `ImportBatchArgs` has `pkColumns` or `pkConstraintName`. We essentially call `tdb.ImportBatch(table, importBatchArgs)`, so it's a bit unexpected that the tdb wouldn't be able to get the pkColumns for the table. Make sense?

Reply:
Yeah, the flow of information seems to be messed up a bit.

I have noted this down, would like to merge this merge as it has been pending from some time.

Is it ok if i take this in a separate PR @makalaaneesh ?

Reply:
Yeah I had approved the PR already 👍